server/konnectors/podcast.js

```diff
@@ -273,8 +275,8 @@ function createTrackIfNotPresent (trackName, fileID, callback) {
 
     for (const track of tracks) {
```

Comment:
I think the origine of the problem is here : https://github.com/cozy-labs/konnectors/blob/master/server/konnectors/podcast.js#L271

can you `return callback(err)` instead ?

Same here : https://github.com/cozy-labs/konnectors/blob/master/server/konnectors/podcast.js#L226
 Thanks @clochix